src/components/send/SendAssetList.js

```diff
@@ -80,9 +80,18 @@ export default class SendAssetList extends React.Component {
       assets.pop(); // removes not needed coin divider
     }
 
+    //check for placeholder ETH & remove
+    assets[0]?.isPlaceholder ? assets.shift() : null;
```

Comment:
regarding index 0: what happens in the scenario if ETH is pinned/placeholder but there are other assets that are pinned with larger balances (and come in before it?)

Reply:
I added isPlaceholder, it only exists for the hardcoded ETH object(s)

Reply:
what if ETH is not the first item in the assets list?